README.md

```diff
@@ -8,56 +8,70 @@ This system/service/software is not officially supported or endorsed by Glen Dim
 
 ## Quick Start
 
+    import asyncio
     from pynobo import nobo
 
-Either call using the three last digits in the hub serial
+    async def main():
```

Comment:
Should we also give users an example of not using the asyncio? With the async wrappers the following example:
```
import time
from pynobo import nobo

def main():
    # Either call using the three last digits in the hub serial
    hub = nobo('123') 
    # or full serial and IP if you do not want to discover on UDP:
    hub = nobo('123123123123', '10.0.0.128', False)

    # Connect to the hub
    hub.start()

    # Inspect what you get
    def update(hub):
        print(hub.hub_info)
        print(hub.zones)
        print(hub.components)
        print(hub.week_profiles)
        print(hub.overrides)
        print(hub.temperatures)

    # Listen for data updates - register before getting initial data to avoid race condition
    hub.register_callback(callback=update)

    # Get initial data
    update(hub)

    # Hang around and wait for data updates
    time.sleep(60)

    # Stop the connection
    hub.stop()
    time.sleep(1)

main()
```
is very-very close to running  cleanly... Currently it throws a `RuntimeWarning: coroutine 'nobo.start' was never awaited...` (no surprise, since there is not async wrapper for this), and same for nobo.stop. It also hangs at exit, see other comment

Reply:
I agree we should have a synchronous example!

`start()` and `stop()` are new methods, so I did not provide synchronous wrappers for those. If you just remove those your example works.

If `loop` is not provided, the `__init__` will start the thread, to maintain backwards compatibility with the current version. I'm happy to change this, but then we should bump the major version (breaking API change). Perhaps the major version should be bumped anyways with this change.

Reply:
Of course, yes! That works :) Even easier...